### PR TITLE
Index table styles

### PR DESCRIPTION
**Pull request by ninariccimarie**

### CODE REVIEW

File: src/@next/IndexTable/IndexTableStyle.ts [hunk 1]
```diff
@@ -0,0 +1,1745 @@
...
+  .Polaris-Scrollable--hasBottomShadow {
+    box-shadow: var(--pc-scrollable-shadow-bottom);
+  }
+
+  .Polaris-Scrollable--hasTopShadow.Polaris-Scrollable--hasBottomShadow {
+    box-shadow: var(--pc-scrollable-shadow-top),
+      var(--pc-scrollable-shadow-bottom);
+  }
+
+  .Polaris-Popover {
```
Comment (marthayaputra) [inline src/@next/IndexTable/IndexTableStyle.ts:149]: Do we still need Popover styles here, as they have been added in PopoverStyle?

File: src/@next/IndexTable/IndexTableStyle.ts [hunk 2]
```diff
@@ -0,0 +1,1745 @@
...
+    --pc-inline-gap-lg: initial;
+    --pc-inline-gap-xl: initial;
+    gap: var(--pc-inline-gap-xs);
+    display: flex;
+    flex-wrap: var(--pc-inline-wrap);
+    align-items: var(--pc-inline-block-align);
+    justify-content: var(--pc-inline-align);
+  }
+
+  @media (min-width: 30.625em) {
```
Comment (marthayaputra) [inline src/@next/IndexTable/IndexTableStyle.ts]: Do we need this? Also same for all media queries

Comment (ninariccimarie) [reply src/@next/IndexTable/IndexTableStyle.ts]: ohh thanks for spotting this! yes, we can remove 👌

File: src/@next/IndexTable/IndexTableStyle.ts (hunk 1 above)
Comment (ninariccimarie) [reply src/@next/IndexTable/IndexTableStyle.ts:149]: hmm.. seems this one is breaking 🤔 
Screenshot 2023-02-03 at 4 42 13 PM

**Review by marthayaputra — APPROVED** [review]
LGTM

**Pull request merged**